docker/my-dojo/.env

```diff
@@ -53,6 +53,7 @@ BITCOIND_LINUX_GID=1108
 
 BITCOIND_DNSSEED=0
 BITCOIND_DNS=0
+BITCOIND_BLOCKS_DIR=/dev/null
```

Comment:
This actually belongs in docker-bitcoind.conf.tpl file. Inytead of `/dev/null` I would leave the default value empty and test for non-empty value in `restart.sh`.

Reply:
Using docker-bitcoind.conf.tpl it can't create the volume inside the docker file. The variable is always empty if it's outside the .env file

Reply:
That's because you need to manually expose conf variables that you want to be used by docker compose. Example here: https://github.com/Dojo-Open-Source-Project/samourai-dojo/blob/develop/docker/my-dojo/dojo.sh#L41